scalableminds/webknossos
# Build STL in chunks when exporting them

### URL of deployed dev instance (used for testing):
- https://chunkedstlexport.webknossos.xyz

### Steps to test:
- open the l4 dataset and load precomputed meshes
- export them as STL
- import them in another software
- I also downloaded a mesh on the master branch and then diffed the STL via `diff` to ensure that there is no difference

### Issues:
- fixes #7049

------
(Please delete unneeded items, merge only when none are left open)
